.scalafmt.conf

```diff
@@ -0,0 +1,51 @@
+version = 3.5.9
+runner.dialect = scala213
+style = defaultWithAlign
+maxColumn = 120
+lineEndings = unix
+importSelectors = singleLine
+project {
+  git = true
+}
+align = none
+align {
+  tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"}, "<-", "->", "%", "%%" ]
+  arrowEnumeratorGenerator = true
+  openParenCallSite = false
+  openParenDefnSite = false
+}
+binPack {
+  parentConstructors = true
+}
+continuationIndent {
+  callSite = 2
+  defnSite = 2
+}
+newlines {
+  penalizeSingleSelectMultiArgList = false
+  sometimesBeforeColonInMethodReturnType = true
+}
+rewrite {
+  rules = [PreferCurlyFors, RedundantBraces, RedundantParens, AsciiSortImports]
+  redundantBraces {
+    maxLines = 100
+    includeUnitMethods = true
+    stringInterpolation = true
+  }
+}
+spaces {
+  inImportCurlyBraces = false
+  beforeContextBoundColon = false
+}
+rewriteTokens {
+  "\t": "  "
+  "→": "->"
+  "←": "<-"
+  "⇒": "=>"
+}
+
+unindentTopLevelOperators = true
```

Comment:
![image](https://github.com/hmrc/income-tax-property-frontend/assets/56151567/e56740ce-8bf3-4726-8bda-0fb38de7d72e)

Reply:
Valid value for an older version.